xml/System.Text.RegularExpressions/RegexOptions.xml

```diff
@@ -117,7 +117,7 @@
       </ReturnValue>
       <MemberValue>8</MemberValue>
       <Docs>
-        <summary>Specifies that the regular expression is compiled to an assembly. This yields faster execution but increases startup time. This value should not be assigned to the <see cref="P:System.Text.RegularExpressions.RegexCompilationInfo.Options" /> property when calling the <see cref="M:System.Text.RegularExpressions.Regex.CompileToAssembly(System.Text.RegularExpressions.RegexCompilationInfo[],System.Reflection.AssemblyName)" /> method. For more information, see the "Compiled Regular Expressions" section in the <see href="/dotnet/standard/base-types/regular-expression-options">Regular Expression Options</see> article.</summary>
+        <summary>Specifies that the regular expression will compile IL code on the fly, not compile time, runtime to pass string for that specific expression. This value should not be assigned to the <see cref="P:System.Text.RegularExpressions.RegexCompilationInfo.Options" /> property when calling the <see cref="M:System.Text.RegularExpressions.Regex.CompileToAssembly(System.Text.RegularExpressions.RegexCompilationInfo[],System.Reflection.AssemblyName)" /> method. For more information, see the "Compiled Regular Expressions" section in the <see href="/dotnet/standard/base-types/regular-expression-options">Regular Expression Options</see> article.</summary>
```

Comment:
> This yields faster execution but increases startup time

I think this is still important to keep in the docs.

Reply:
But tests don't prove it. This flag does not speed up execution, but on the contrary slows it down.

Reply:
It does speed up execution. You just need to cache the Regex instance, and not create a new instance every time you call it. That's what it means by "increases startup time".

Reply:
> It does speed up execution. You just need to cache the Regex instance, and not create a new instance every time you call it. That's what it means by "increases startup time".

In that case
this line - "This yields faster execution but increases startup time"
should be replaced by this - "This yields faster execution only if expression cached, but increases startup time"